src/rust-crypto/rust-crypto.ts

```diff
@@ -583,9 +583,30 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
      * @returns the VerificationRequest that is in progress, if any
      *
      */
-    public findVerificationRequestDMInProgress(roomId: string): undefined {
-        // TODO
-        return;
+    public findVerificationRequestDMInProgress(roomId: string): VerificationRequest | undefined {
+        const room = this.store.getRoom(roomId);
+
+        if (!room) throw new Error(`unknown roomId ${roomId}`);
+
+        const members = room.getMembers();
```

Comment:
Ironically, this method is only called in one place, where we know *exactly* which user we are interested in:

https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/verification.ts#L123

Let's: 
 * add a second optional parameter `userId?: string` to  `findVerificationRequestDMInProgress`. If both `roomId` and `userId` are provided, `findVerificationRequestDMInProgress` only returns requests in that room for that user.
 * Update legacy `Crypto.findVerificationRequestDMInProgress` accordingly
 * deprecate calling `findVerificationRequestDMInProgress` without the second parameter
 * update react-sdk to pass the second parameter
 * change this code to just `raise new Error` if the second parameter is not passed
 * get rid of the IStore awfulness
 * profit